docs/conf.py

```diff
@@ -1,8 +1,13 @@
 import re
+import sys
+import os
+
 
 project = "nextcord-ext-ipc"
-copyright = "2021, Nextcord Developers"
-author = "Nextcord Developers"
+copyright = "2021, nextcord"
+author = "nextcord"
+
+sys.path.insert(0, os.path.abspath('..'))
```

Comment:
Black wants double quotes here
```suggestion
sys.path.insert(0, os.path.abspath(".."))
```